court_scraper/base/selenium_site.py

```diff
@@ -7,33 +7,31 @@
 
 class SeleniumSite:
 
-
     def _init_chrome_driver(self, headless=True):
         chrome_options = self._build_chrome_options(headless=headless)
         executable_path = shutil.which('chromedriver')
         driver = webdriver.Chrome(options=chrome_options, executable_path=executable_path)
         return driver
 
-    def _build_chrome_options(self, headless=True):
+    def _build_chrome_options(self, headless=True, random_user=False):
```

Comment:
These changes provide an option to platforms to use a random_user if it suits the scraping need. I had to modify it for Wisconsin because the ua would sometimes be and out-of-date browser that the court website thought would not support the captcha solver.

Reply:
I coded a default option that disables it but we could easily choose the other option.